### ISSUE TITLE
OpenUPM glTFast does not work with *KTX for Unity* package

### ISSUE THREAD

**Issue by Ivolutio** [atteneder/glTFast#732]
**Describe the bug**
I cloned this repository (I tried openupm and main branche, as well as downloading the source code of the 6.8.0 version release. Made sure I installed Draco for Unity and KTX for Unity through the Unity packages. I try to import a .glb file on runtime or in-editor and it prompts me with `KTX for Unity package needs to be installed in order to support glTF extension KHR_texture_basisu!`.

**Files**

Attached is the manifest from my packages folder.
[manifest.json](https://github.com/user-attachments/files/17955732/manifest.json)

**Desktop (please complete the following information):**
 - glTFast version: latest commit from openupm branch, latest commit from main branch, v6.8.0
 - Unity Editor version: 2022.3.46f1
 - Render Pipeline and version: URP 14.0.11
 - Platform: Editor, Runtime Android

**Additional context**
The package works correctly if I install it through the Unity package registry, but I want to try play with the source code.

**Comment by atteneder** [comment]
Hi @Ivolutio ,

Thanks for pointing it out!

TL;DR: If you want to work on the source, fork the [Unity fork](https://github.com/unity-Technologies/com.unity.cloud.gltfast) instead.

The issue is that the `openupm` branch still uses/depends on the OpenUPM version of *KTX for Unity* (*KtxUnity*). I haven't been able to update those in a while though, so this needs a solution.

Thanks!

**Comment by atteneder** [comment]
[glTFast 6.9.0](https://github.com/atteneder/glTFast/blob/openupm/CHANGELOG.md#changed-3) (the original version, `openupm` branch, available via OpenUPM) now supporte KTX for Unity (by Unity).
